pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -118,19 +134,42 @@ func (o *MustGatherOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, arg
 	return nil
 }
 
-func (o *MustGatherOptions) resolveMustGatherImage() (string, error) {
+func (o *MustGatherOptions) resolveImageStreamTagString(s string) (string, error) {
+	namespace, name, tag := parseImageStreamTagString(s)
+	if len(namespace) == 0 {
+		namespace = "openshift"
```

Comment:
if it doesn't change an existing CLI contract, lets not default at all and simply fail instead.

Reply:
Done